frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -465,6 +482,27 @@ export const SaveAsDefaultCheckbox: React.FC<SaveAsDefaultCheckboxProps> = ({
   );
 };
 
+export const SendResolvedAlertsCheckbox = ({ formField, formValues, dispatchFormChange }) => {
+  return (
+    <label className="co-no-bold" htmlFor={formField}>
+      <input
+        type="checkbox"
+        id={formField}
+        data-test-id="send-resolved-alerts"
+        onChange={(e) =>
+          dispatchFormChange({
+            type: 'setFormValues',
+            payload: { [formField]: e.target.checked },
+          })
+        }
+        checked={formValues[formField]}
+        aria-checked={formValues[formField]}
+      />
+      &nbsp; Send resolved alerts to this receiver?
```

Comment:
`&nbsp;` shouldn't be needed if you use the checkbox component or Bootstrap checkbox classes.